docs/content/guides/cell-features/conditional-formatting/angular/example1.ts

```diff
@@ -1,7 +1,7 @@
 /* file: app.component.ts */
 import { Component } from '@angular/core';
 import { GridSettings } from '@handsontable/angular-wrapper';
-import Handsontable from 'handsontable';
+import Handsontable from 'handsontable/base';
 import { registerRenderer } from 'handsontable/renderers';
 
```

Comment:
Missing `registerAllModules()` call?

Reply:
It's already present lower in the file https://github.com/handsontable/handsontable/blob/721d34abca28453404e497fa7276d9d622e4d526/docs/content/guides/cell-features/conditional-formatting/angular/example1.ts#L133-L134